runtime/doc/api.txt

```diff
@@ -979,6 +979,19 @@ nvim_get_hl_id_by_name({name})                      *nvim_get_hl_id_by_name()*
 
     similar to |hlID()|, but allocates a new ID if not present.
 
+nvim_get_hl_ns({*opts})                                     *nvim_get_hl_ns()*
```

Comment:
In https://github.com/neovim/neovim/pull/24390#issuecomment-1643966167 there was an open question about naming this `nvim_get_ns`. 

Why do we need highlight-specific namespace functions?

Reply:
Strictly speaking, we didn't "need" highlight namespaces to begin with, we added them because it was something we wanted (it is a little fun).

The more complex involved answer is, if we had window-local options which actually worked (with a mode where they're actually being local to the window, which I plan to work on "soon"), then `hl_ns` could just have been yet another global-local option. But in the meanwhile, having API functions ilke `nvim_set/get_hl_ns` to get and set the current highlight namespace seems like the best option. I don't see how they generalize to other uses of namespaces, and thus removing "hl" out of the the name would only hurt discoverability and understanding of code.